ragstack-e2e-tests/e2e_tests/test_utils/astradb_vector_store_handler.py

```diff
@@ -201,23 +201,19 @@ def try_delete_with_backoff(self, collection: str, sleep=1, max_tries=5):
             self.try_delete_with_backoff(collection, sleep * 2, max_tries)
 
     def ensure_astra_env_clean(self, blocking=False):
-        logging.info("Ensuring astra env is clean")
-        self.delete_collection_handler.run_delete(self.astra_ref.collection)
```

Comment:
Actually, was this (incorrectly) deleting the collection we just created for the purposes of the test?

And later on when we reassigned the `self.astra_ref.collection = .. `, how is that new collection being created?